technology/jackson/src/main/java/com/github/mizool/technology/jackson/ClientFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.github.mizool.technology.jackson;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+
+import lombok.experimental.UtilityClass;
+
+import org.glassfish.jersey.jackson.JacksonFeature;
+
+@UtilityClass
+class ClientFactory
```

Comment:
`Client` sounds too unspecific to me. How about prefixing this and the related classes with `JaxRs`? Alternatively, one could move these Jax-RS related Jackson support classes to a new package `com.github.mizool.technology.jackson.jaxrs`. I think I favor the latter.